cmd/protoc-gen-gno/main.go

```diff
@@ -156,3 +161,18 @@ func generateJSONUtils(gen *protogen.Plugin, file *protogen.File) {
 		g.P("}")
 	}
 }
+
+func genImportStmts(gen *protogen.Plugin, g *protogen.GeneratedFile, file *protogen.File) {
+	for i, imps := 0, file.Desc.Imports(); i < imps.Len(); i++ {
+		imp := imps.Get(i)
+		impFile, ok := gen.FilesByPath[imp.Path()]
+		if !ok {
+			continue
+		}
+
+		alias := g.QualifiedGoIdent(impFile.GoImportPath.Ident(""))
```

Comment:
calling QualifiedGoIdent on the imported types should add the import statement automatically IIRC